src/views/containerNode.ts

```diff
@@ -15,23 +15,36 @@ export class ContainerNode extends DataNode {
         super(nodeData, parent);
     }
 
+    private _containerType: ContainerType;
```

Comment:
It's not used for now, how about removing it?

Reply:
I was thinking to leave it as it will "cache" the outcome of the **getContainerType()** method. Does it make sence then to still be present?

Reply:
OK I see. Yeah it makes sense!